src/api/getArticle.js

```diff
@@ -85,6 +93,31 @@ const fixImageUrl = (htmlString) => {
   return htmlString.replace(/src="\/\//gi, 'src="https://')
 }
 
+const fixTableCaption = (htmlString, moreInformationText) => {
+  const hiddenClassName = 'hidden-in-table'
+  const parser = new DOMParser()
+  const node = parser.parseFromString(htmlString, 'text/html')
+  const tableNodes = node.querySelectorAll('table.wikitable, table.multicol')
+  for (const tableNode of tableNodes) {
+    const thContent = Array.from(tableNode.querySelectorAll('th')).map(th => th.textContent).join(', ')
```

Comment:
Is this `th` as in table header content?

Reply:
yes